ui/src/main/webapp/src/app/misc/about.component.html

```diff
@@ -0,0 +1,94 @@
+                            </p>
+                            <p>
+                                With the Red Hat Application Migration Toolkit command line interface (CLI), you can create migration reports from the terminal.
+                                To get started using the CLI, visit <a href="https://access.redhat.com/documentation/en/red-hat-jboss-migration-toolkit/">Red Hat Application Migration Toolkit Command Line Interface Console documentation</a>.
+                            </p>
+
+                            <h2>Links</h2>
+
+                            <dl class="dl-horizontal">
+                                <dt>Follow on Twitter</dt>
+                                <dd><a href="http://twitter.com/jbosswindup">@jbossWindup</a></dd>
+
+                                <dt>Website</dt>
+                                <dd><a href="http://windup.jboss.org">http://windup.jboss.org</a></dd>
+
+                                <dt>Documentation</dt>
+                                <dd><a href="https://access.redhat.com/documentation/en/red-hat-jboss-migration-toolkit/">https://access.redhat.com/documentation/en/red-hat-jboss-migration-toolkit/</a></dd>
+                            </dl>
+
+                            <dl class="dl-horizontal">
+                                <dt>GitHub Source</dt>
+                                <dd><a href="https://github.com/windup/windup">https://github.com/windup/windup</a></dd>
+
+                                <dt>GitHub Wiki</dt>
+                                <dd><a href="https://github.com/windup/windup/wiki">https://github.com/windup/windup/wiki</a></dd>
+
+                                <dt>Discussion Forum</dt>
+                                <dd><a href="https://community.jboss.org/en/windup?view=discussions">https://community.jboss.org/en/windup</a></dd>
+
+                                <dt>Mailing List</dt>
+                                <dd><a href="https://lists.jboss.org/mailman/listinfo/windup-dev">https://lists.jboss.org/mailman/listinfo/windup-dev</a></dd>
+
+                                <dt>Issues Tracking</dt>
+                                <dd><a href="https://issues.jboss.org/browse/WINDUP">https://issues.jboss.org/browse/WINDUP</a></dd>
+                            </dl>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-md-12">
+                            <h2>Contributors</h2>
+
+                            <div id="windup-contributors"></div>
+                            <p class="windup-contributors small">A special <strong>thanks</strong> to our contributors!</p>
+                        </div>
+
+                        <script type="text/javascript">
+                            var divTarget = $("#windup-contributors");
```

Comment:
I believe that he copied it from windup core for expedience. Technically he had marked this as WIP, I just merged it prematurely so that we wouldn't get an error when clicking "About". :)

+1 to fixing it whenever we have time.